docs/azure/sdk/dependency-injection.md

```diff
@@ -31,34 +31,38 @@ For demonstration purposes, the sample code in this article uses the Key Vault S
 ```dotnetcli
 dotnet add package Azure.Security.KeyVault.Secrets
 dotnet add package Azure.Storage.Blobs
+dotnet add package Azure.Messaging.ServiceBus
 ```
 
-## Register clients
+## Register clients and subclients
```

Comment:
I believe there are other docs linking to the old section. Those will need to be updated too.

Reply:
Yes I think those are the errors in the build report, this is still a draft so I'm fixing some things like that, but the content itself is generally ready for review.